src/controllers/vscodeWrapper.ts

```diff
@@ -339,6 +339,13 @@ export default class VscodeWrapper {
     }
 
     /**
+     * Change a configuration setting
+     */
+    public setConfiguration(extensionName: string, resource: string, value: any): Thenable<void> {
```

Comment:
It'd be good to update https://github.com/microsoft/vscode-mssql/blob/2e745d3d24344216b453c9f8db334775a9bae2a4/src/connectionconfig/connectionconfig.ts#L153 to use this as well